package.json

```diff
@@ -4,17 +4,18 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "npm run build:clean && npm run build:esm && npm run build:cjs && npm run build:tailwind",
+    "build": "npm-run-all build:**",
+    "build:clean": "rm -rf ./lib",
     "build:esm": "tsc -p ./tsconfig.esm.json",
     "build:cjs": "tsc -p ./tsconfig.cjs.json && mv lib/cjs/components/index.js lib/cjs/components/index.cjs",
     "build:tailwind": "npx tailwindcss -i ./app/globals.css -o ./lib/globals.css --minify && cp ./tailwind.config.ts ./lib",
-    "build:clean": "rm -rf ./lib",
-    "start": "next start",
+    "start": "storybook dev -p 6006",
```

Comment:
Much better to start storybook instead of the not needed nextJS part of this project